packages/server/graphql/queries/helpers/makeUpcomingInvoice.ts

```diff
@@ -35,7 +35,11 @@ export default async function makeUpcomingInvoice(
   const subscription = stripeInvoice.lines.data.find(
     ({plan}) => plan?.id === StripeManager.PARABOL_TEAM_600
   )
-  if (subscription && subscription.quantity !== quantity) {
+  if (
+    subscription &&
+    subscription.plan?.tiers_mode !== 'volume' &&
```

Comment:
+1 should not be necessary as we're explicitly searching for `PARABOL_TEAM_600`.
I think I would like it more if both places are consistent, i.e. we check for `id === PARABOL_TEAM_600` in both or for the `tiers_mode`